src/sentry/api/serializers/models/group_stream.py

```diff
@@ -156,6 +160,34 @@ def get_stats(
             return self.query_tsdb(item_list, query_params, user=user, **kwargs)
 
 
+# Serializer for User Feedback
+# Used to expose the linked integration issues for a list of feedbacks (groups)
+class ExternalIssueSerializer(Serializer):
+    def get_attrs(self, item_list: List[ExternalIssue], user: User, **kwargs: Any):
+        group_linked_issues = {}
```

Comment:
i think the convention for `get_attrs` is to simply name this variable `result`. https://github.com/getsentry/sentry/blob/d55f0729afd34f080c83c2bb5ffc0aec4887d92d/src/sentry/api/serializers/models/deploy.py#L16